hr_expense_advance_clearing/views/hr_expense_views.xml

```diff
@@ -0,0 +1,103 @@
+<odoo>
+    <record id="hr_expense_sheet_view_search" model="ir.ui.view">
+        <field name="name">hr.expense.sheet.view.search</field>
+        <field name="model">hr.expense.sheet</field>
+        <field name="inherit_id" ref="hr_expense.hr_expense_sheet_view_search" />
+        <field name="arch" type="xml">
+            <separator position="after">
+                <filter
+                    string="Advance"
+                    name="advance_expense"
+                    domain="[('advance', '=', True)]"
+                />
+                <filter
+                    string="Advance (not cleared)"
+                    name="advance_expense_uncleared"
+                    domain="[('advance', '=', True), ('residual', '>', 0.0)]"
+                />
+                <filter
+                    string="Clearing"
+                    name="advance_clearing"
+                    domain="[('advance_sheet_id', '!=', False)]"
+                />
+            </separator>
+        </field>
+    </record>
+    <record id="action_hr_expense_sheet_advance_clearing" model="ir.actions.act_window">
+        <field name="name">Advance Clearing</field>
+        <field name="res_model">hr.expense.sheet</field>
+        <field name="view_mode">form</field>
+        <field name="search_view_id" ref="hr_expense_sheet_view_search" />
+        <field
+            name="domain"
+        >[('employee_id.user_id', '=', uid), ('state', '!=', 'cancel')]</field>
```

Comment:
```suggestion
        <field name="domain">
            [('employee_id.user_id', '=', uid), ('state', '!=', 'cancel')]
        </field>
```
I think this format is better. What do you think about it?

Reply:
Done.